app/assets/stylesheets/spotlight/_header.scss

```diff
@@ -143,3 +143,28 @@
     width: 1px;
   }
 }
+
+.link-unstyled {
+  &,
+  &:visited,
+  &:hover,
+  &:active,
+  &:focus,
+  &:active:hover {
+    font-style: inherit;
```

Comment:
Why do we need to change anything beyond `font-weight`, `color` or `text-decoration`?

Reply:
Nope